packages/webview_flutter/webview_flutter_android/lib/src/android_webview.dart

```diff
@@ -66,6 +66,91 @@ class WebView {
     return api.setWebContentsDebuggingEnabled(enabled);
   }
 
+  /// Loads the given data into this WebView using a 'data' scheme URL.
+  ///
+  /// Note that JavaScript's same origin policy means that script running in a
+  /// page loaded using this method will be unable to access content loaded
+  /// using any scheme other than 'data', including 'http(s)'. To avoid this
+  /// restriction, use [loadDataWithBaseURL()] with an appropriate base URL.
+  ///
+  /// The [encoding] parameter specifies whether the data is base64 or URL
+  /// encoded. If the data is base64 encoded, the value of the encoding
+  /// parameter must be `'base64'`. HTML can be encoded with
+  /// `base64.encode(bytes)` like so:
+  /// ```dart
+  /// import 'dart:convert';
+  ///
+  /// final unencodedHtml = '''
+  ///   <html><body>'%28' is the code for '('</body></html>
+  /// ''';
+  /// final encodedHtml = base64.encode(utf8.encode(unencodedHtml));
+  /// print(encodedHtml);
+  /// ```
+  ///
+  /// The [mimeType] parameter specifies the format of the data. If WebView
+  /// can't handle the specified MIME type, it will download the data. If
+  /// `null`, defaults to 'text/html'.
+  Future<void> loadData(String data, String? mimeType, String? encoding) {
+    return api.loadDataFromInstance(
+      this,
+      data,
+      mimeType ?? _nullStringIdentifier,
+      encoding ?? _nullStringIdentifier,
+    );
+  }
+
+  /// Loads the given data into this WebView, using baseUrl as the base URL for
```

Comment:
This should all be on one line. I personally don't mind if the first line exceeds the line limit, but `...using baseUrl as the base URL for the content.` could probably be separated to avoid line limit if that was your intention.

Reply:
Fixed.